events/wmt08.md

```diff
@@ -89,68 +89,40 @@ The **Third Workshop on Machine Translation** (**WMT08**) took place on 19 June,
 
 Full results of the shared tasks: [*Further Meta-Evaluation of Machine Translation*](https://www.statmt.org/wmt08/pdf/WMT09.pdf)
 
-### News translation
+### Translation
 
-The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` ("greater than or equal to others") score.
+The results were determined with a [relative ranking](wmt.md#relative-ranking), the `≥ others` (“greater than or equal to others”) score, [constituent ranking](wmt.md#constituent-ranking), and [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement).
 
-It measures how often a system was judged to be better than or equal to any other system.
+The `≥ others` score measures how often a system was judged to be better than or equal to any other system.
 
-#### → English
-
-| Language pair | System | `≥ others` |
-| --- | --- | --- |
-| Czech → | `DCU` | 0.675 | 
-| French → | `LIUM-SYS-CONTR` | 0.693 | 
-| German → | `RBMT2` | 0.742 | 
-| Spanish → | `RBMT4` | 0.768 | 
-| Hungarian → | `MORPHOLOGIC` | 0.853 | 
-
-#### English →
-
-| Language pair | System | `≥ others` |
-| --- | --- | --- |
-| → Czech | `PC-TRANSLATOR` | 0.715 | 
-| → French | `RBMT4` | 0.780 | 
-| → German | `RBMT2` | 0.787 | 
-| → Spanish | `RBMT5` | 0.724 | 
-
-The results were determined with a [constituent ranking](wmt.md#constituent-ranking), the `≥ others` ("greater than or equal to others") score.
-
-#### → English
-
-| Language pair | System | `≥ others` |
-| --- | --- | --- |
-| French → | `LIMSI` | 0.801 | 
-| German → | `SAAR-CONTR` | 0.775 | 
-| Spanish → | `CUED-CONTR` | 0.818 | 
-
-#### English →
-
-| Language pair | System | `≥ others` |
-| --- | --- | --- |
-| → Czech | `PC-TRANSLATOR` | 0.674 | 
-| → French | `RBMT4` | 0.768 | 
-| → German | `RBMT4` | 0.748 | 
-| → Spanish | `RBMT4` | 0.754 | 
-
-The results were determined with a [yes or no constituent judgement](wmt.md#yes-or-no-constituent-judgement), the acceptability score. 
+The constituent score measures how often a system was judged to be better than or equal to any other system in the constituent-based evaluation.
```

Comment:
I fear that only adding the info about `≥ others` and `constituent score` would leave `relative ranking` and `yes-and-no constituent judgement` out.

Perhaps these explanations are not necessary, as the definitions of these scores are already included in `wmt.md`, what do you think?

Reply:
I added them as the WMT article has only the explanations for how those evaluation types work and not how their scores are calculated. I suggest adding the score calculation explanations to the wmt.md as well before cutting those out of this article.

Reply:
Yes, I agree. Let's do that.